pkglistgen/tool.py

```diff
@@ -785,7 +918,27 @@ def update_and_solve_target(
                 for line in sorted(output):
                     f.write(line + '\n')
 
-        self.commit_package(product_dir)
+        if git_url:
+            if os.path.isfile(f"{self.productcompose_dir}/default.productcompose"):
+                logging.debug(subprocess.check_output(
+                    ['git', 'add', self.productcompose_dir, f"{self.productcompose_dir}/default.productcompose"],
+                    cwd=cache_dir, encoding='utf-8'))
+            if product_dir and os.path.isdir(product_dir):
+                logging.debug(subprocess.check_output(
+                    ['git', 'add', product_dir],
+                    cwd=cache_dir, encoding='utf-8'))
+            if os.path.isdir(self.productcompose_dir):
+                logging.debug(subprocess.check_output(
+                    ['git', 'add', self.productcompose_dir],
```

Comment:
This adds all of productcompse_dir, the other git add calls for it are not needed

Reply:
clean'd up